template_feed/Microsoft.DotNet.Test.ProjectTemplates.8.0/content/MSTest-CSharp/.template.config/template.json

```
@@ -0,0 +1,89 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": ["Test", "MSTest"],
+  "name": "MSTest Test Project",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A project that contains MSTest unit tests that can run on .NET Core on Windows, Linux and MacOS.",
+  "groupIdentity": "Microsoft.Test.MSTest",
+  "precedence": "10000",
+  "identity": "Microsoft.Test.MSTest.CSharp.7.0",
+  "shortName": "mstest",
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "sourceName": "Company.TestProject1",
+  "preferNameDirectory": true,
+  "symbols": {
+    "TargetFrameworkOverride": {
+      "type": "parameter",
+      "description": "Overrides the target framework",
+      "replaces": "TargetFrameworkOverride",
+      "datatype": "string",
+      "defaultValue": ""
+    },
+    "Framework": {
+      "type": "parameter",
+      "description": "The target framework for the project.",
+      "datatype": "choice",
+      "choices": [
+        {
+          "choice": "net7.0",
+          "description": "Target net7.0"
+        }
+      ],
+      "replaces": "net7.0",
+      "defaultValue": "net7.0"
+    },
+    "EnablePack": {
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false",
+      "description": "Whether or not to enable packaging (via (\"dotnet pack\") for the project."
+    },
+    "HostIdentifier": {
+      "type": "bind",
+      "binding": "HostIdentifier"
```

Consider using precise syntax for better perf (new feature since .NET 7.0)
```suggestion
      "binding": "host:HostIdentifier"
```

---

Fixed.